scripts/yocto-build-and-test.sh

```diff
@@ -522,6 +523,7 @@ build_and_test_client() {
 
             copy_clean_image "${machine_name}" "${board_name}" "${image_name}" "${device_type}" "$extension"
             copy_clean_image "${machine_name}" "${board_name}" "mender-image-full-cmdline-rofs" "${device_type}" "$extension"
+            copy_clean_image "${machine_name}" "${board_name}" "mender-image-full-cmdline-rofs-commercial" "${device_type}" "$extension"
             copy_clean_image "${machine_name}" "${board_name}" "mender-monitor-image-full-cmdline" "${device_type}" "$extension"
             copy_clean_image "${machine_name}" "${board_name}" "mender-gateway-image-full-cmdline" "${device_type}" "$extension"
```

Comment:
@kacf,

(pre-existing) Shouldn't we loop over `$images_to_build` here? It will fail if these images are not found :thinking: 

I think we have not seen this error because since we refactor this code no new images have been added so that `meta-mender/master` gets out of sync with the supported branches.

Reply:
Yes, I think you're right.

Reply:
Noted: Doing it in a separate commit after just getting a green pipe for the normal stuff.